app-misc/francis/francis-9999.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_CATEGORY="utilities"
+KFMIN=6.5.0
+QTMIN=6.6.0
```

Comment:
We use `6.7.2` as baseline in all other KDE Gear packages, so this should too. The reason for that was yet another ABI break on that release so it was necessary to enforce this to avoid breakages.